src/components/LyricView.tsx

```diff
@@ -33,6 +34,13 @@ const LyricView: React.FC<LyricViewProps> = (props: LyricViewProps) => {
     getSong(props.songNumber, bookId).then((song) => {
       setSong(song);
     });
+    // check for initial screen width, since the listener only checks on resize
+    props.setTooNarrow(window.innerWidth <= 991);
```

Comment:
What happens also if the screen is zoomed out/in a lot? The pixel width may be bigger than 991 and still overlap or smaller and not overlap if the user has the screen zoomed in/out?

I think a better idea might be to check if the left right buttons overlap to the Music/Lyric view. I googled and found these, which might be helpful 

https://stackoverflow.com/questions/12066870/how-to-check-if-an-element-is-overlapping-other-elements

https://codereview.stackexchange.com/questions/237255/most-performant-way-to-check-if-elements-are-overlapping

A reusable way to do this would be to create a new file `utils/UiUtils.tsx` or something which handles UI related helper functions. Keep it generic with just a function like `doElementsOverlap(element1: (some object type that any TS element can pass in), element2): boolean` or something

Reply:
Yea this might be a good idea, I'll look into it